tajo-core/src/main/java/org/apache/tajo/ws/rs/requests/SubmitQueryRequest.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.tajo.ws.rs.requests;
+
+import java.util.Map;
+
+import com.google.gson.annotations.Expose;
+
+public class SubmitQueryRequest {
+
+  @Expose private String query;
+  @Expose private Map<String, String> sessionVariables;
+  
+  public String getQuery() {
+    return query;
+  }
+  public void setQuery(String query) {
+    this.query = query;
+  }
+  public Map<String, String> getSessionVariables() {
```

Comment:
SubmitQueryRequest::getSessionVariables and SubmitQueryRequest:: setSessionVariables seem to be unused methods.